packages/devtools/lib/src/main.dart

```diff
@@ -136,6 +140,44 @@ class PerfToolFramework extends Framework {
     screensReady.future.then(app.devToolsReady);
   }
 
+  void initSseConnection() {
```

Comment:
this name is too generic. We use SSE to connect to the VMService as well.
Maybe make this method:
`initDevtoolsServerConnection()` instead